react-app/src/face-rec.js

```diff
@@ -151,6 +151,7 @@ function removeFileExt (fileName) {
 async function loadReferenceImages (paths) {
     const descriptors = []
 
+    // For reference images already in public folder
```

Comment:
You can just delete this section and the images from `public/images`. This should actually speed up the recognition as well